codecov_auth/admin.py

```diff
@@ -89,6 +89,34 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+@admin.register(SentryUser)
+class SentryUserAdmin(AdminMixin, admin.ModelAdmin):
+    list_display = (
+        "name",
+        "email",
+    )
+    search_fields = (
+        "name__iexact",
+        "email__iexact",
```

Comment:
This probably won't scale well since we don't have indexes on these fields but probably fine for now

Reply:
Yea that's true. A scalability issue to query the sentry_user table would be a great problem to have!